src/Schema/PgsqlColumnSchema.php

```diff
@@ -8,15 +8,16 @@
 use Yiisoft\Db\Expression\ExpressionInterface;
 use Yiisoft\Db\Expression\JsonExpression;
 use Yiisoft\Db\Pgsql\Query\ArrayParser;
-use Yiisoft\Db\Schema\ColumnSchema as AbstractColumnSchema;
+use Yiisoft\Db\Schema\ColumnSchema;
 
 use function array_walk_recursive;
+use function in_array;
 use function is_array;
 use function is_string;
 use function json_decode;
 use function strtolower;
 
-class ColumnSchema extends AbstractColumnSchema
+final class PgsqlColumnSchema extends ColumnSchema
```

Comment:
Isn't namespace enough to tell that the `ColumnSchema` belongs to PostgreSQL?

Reply:
FIxed.